### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by OnlyUnknown**
- Added style to the navigator
- Added style to the book container
- Added style to the add book form
- Linters has been added

### CODE REVIEW

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Required Changes 🔁
## Hello @OnlyUnknown  👋 🙂

You're doing a great job on your project. We're proud of you. Good job so far! 

It's OK to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job. 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of GitHub Flow
  ✅  Descriptive Pull Request
  ✅  No linter errors

## Required Changes ♻️

- [ ] Please write down the steps in the README file, how can someone run your project. As you used Vite to create this project and there's no Start script in the package.json file.
I was trying 
```
npm run start
```
which didn't work because you're using Vite. To make sure no one else faces this issue. you can write steps on how to run this project like
```
npm run dev
```
---
- [ ] The website isn't responsive at this moment because there's a horizontal scroll bar. Kindly work on it and remote it as you can in the screenshot. 👍 

Screenshot 2023-05-19 at 6 48 19 PM

----
- [ ] Kindly make sure you're using the same border color and border width on the Book Card as on the zepline design. As you can see there's a difference between your design and the zepline design. 
Also, Comment, Edit, Delete button has a lot of space in your design. kindly make sure it matches with the Zepline design.

### [YOUR DESIGN]

Screenshot 2023-05-19 at 6 50 32 PM

### [ZEPLINE DESIGN]
Screenshot 2023-05-19 at 6 53 13 PM

----
- [ ] Input fields sizes aren’t looking consistent. One is bigger and the other one is smaller. Also, the fields and button should be on the same line according to the Zepline design. Kindly make sure it matches with required design 👍 

### [YOUR DESIGN]
Screenshot 2023-05-19 at 6 55 30 PM

### [ZEPLINE DESIGN]
Screenshot 2023-05-19 at 6 56 18 PM




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tugsem** [comment]
## STATUS: REQUIRED CHANGES ♻️
Hello @OnlyUnknown 👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text

 ## Highlights 🚀
✔️Good job on aligning button next to the form area.

 ## Required Changes ♻️

- [x] As the prev reviewer required, you need to update your readme file to show the right command to start the project. 
- [ ] As per the requirement, please use the same border colors on your card containers. Your border is darker than the given design. You can check the screenshots attached by the prev reviewer above.
image
- [ ] Please remove horizontal scrolling from your body element as the screen width gets smaller it appears on my screen and it doesn't seem responsive. Also, the button and the logo shouldn't collapse according to the given design. 
- [ Optional ] Remove the hardcoded ';' from your HTML as I circled them below. The prev reviewer hasn't addressed but it is highly recommended for a better design.
image
- [ Optional ] In my opinion, the prev reviewer should have invalidated this project because you didn't use react-scripts and built the project with vite. In your following projects and for this project I suggest you use react-scripts which runs the app with npm start as we expected. 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css [hunk 1]
```diff
@@ -90,30 +92,32 @@ a:active { text-decoration: none; }
 .icon {
   width: 2.813rem;
   height: 2.813rem;
-  margin: 0 0 0 41.5rem;
+  margin-left: auto;
   padding: 0.875rem;
   border: solid 1px var(--white);
 }
 
 .BookC {
```
Comment (topeogunleye) [inline src/App.css:100]: - [ ] As suggested by the previous reviewers, the borders are not the same color as compared to the the Given design on Zeplin:

| Your Design  |  Zeplin Design |
| ------------- | ------------- |
| image  | image |
| Content Cell  | Content Cell  |

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @OnlyUnknown, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Remove horizontal scrolling from your body element ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._
- [ ] As suggested by the previous reviewers, Kindly update your README file to have `npm run dev` and not `npm run start` as your run command for your application. This is because you're using vite and `npm run start` will not start the application.


## EXAMPLE
### Usage

To run the project, execute the following command:


- Start the development server
```.md
npm run dev
```
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
- [ ] In future pull requests, consider following the recommended branch naming conventions, such as using branch names that reflect the specific feature or task being implemented. This will enhance code organization and collaboration within the team. For example: `feature/added-styling`.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Navigation.jsx [hunk 2]
```diff
@@ -1,12 +1,16 @@
 import { Link } from "react-router-dom"
+import '../App.css'
 
 const Nav = () => {
```
Comment (Ismailco) [inline src/components/Navigation.jsx:4]: - [ ] [OPTIONAL] Kindly make sure to follow the provided design, the profile icon should be in blue and the yellow line at the bottom of the nav bar should not be present in your app

Screenshot_20230520_155742

File: src/components/AddForm.jsx [hunk 3]
```diff
@@ -17,11 +17,12 @@ const handleClick2 = () => {
 }
 
   
-    return<>
+    return<div className='addC'>
```
Comment (Ismailco) [inline src/components/AddForm.jsx:20]: - [ ] [OPTIONAL] Kindly make sure the `Add New Book` form is taking the whole space in the right to match the book's container. Also, the page components should be centered kindly make sure to fix this so your design reflects the provided design rules

Screenshot_20230520_155727

**Review by Ismailco — APPROVED** [review]
### Hi @OnlyUnknown, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**